src/efi.rs

```diff
@@ -9,10 +9,10 @@ use std::io::prelude::*;
 use std::path::Path;
 use std::process::Command;
 
-use anyhow::{bail, Result};
+use anyhow::{bail, Context, Result};
 use serde::{Deserialize, Serialize};
 
-use chrono::NaiveDateTime;
+use chrono::prelude::*;
```

Comment:
nit: This module is meant to be used as a `*` import. However, it would be nice to denote this and maybe link to [the doc](https://docs.rs/chrono/0.4.15/chrono/prelude/index.html) in a comment.

Reply:
The `prelude` naming is a pretty strong Rust convention for "use this for `*` imports" I think.